src/views/Home/components/PrizeTracks/PrizeTracks.scss

```diff
@@ -0,0 +1,107 @@
+@use "~stylesheets/typography";
+
+.prizeTracks {
+  background: linear-gradient(#b0b6f7, #b0b6f7);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  &__title {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    &__container {
+      width: 524px;
+      max-width: 90%;
+      height: 106px;
+      background-color: #32578E;
+      border-radius: 28px;
+      display: flex;
+      justify-content: center;
+      align-items: center; 
+      &--header {
+        @include typography.title;
+        font-size: 70px;
+        text-shadow: 5px 0px #777CE7;
+      }
+    }
+  }
+  
+  &__images {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    max-width: 1600px;
+    align-items: center;
+    &__container {
+      margin: 40px 50px;
+      width: 341px;
+      min-width: 260px;
+      aspect-ratio: 1/1;
+      border-radius: 50%;
+      background-color: rgba(255, 255, 255, 0.6);
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      position: relative;
+      box-shadow: -4px 8px 4px rgba(20, 28, 94, 0.4); 
+      &--hover {
+        background-color: #162C7B;
+        opacity: 0.76;
+        color: white;
+        border-radius: 50%;
+        width: 100%;
+        aspect-ratio: 1/1;
+        position: absolute;
+        z-index: 3;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        @include typography.title;
+        font-style: italic;
+        text-align: center;
+      }
+      &--image {
+        text-align: center;
+      }
+    }
+  }
+}
+
+.svg-shadow {
+  filter: drop-shadow(0px 5px 5px rgb(0 0 0 / 0.4));
+}
+
+.track-img-dimensions {
+  width: 75%;
+}
+
+.track-img-dimensions-golden {
+  width: 85%;
+}
+
+.track-img-dimensions-earth {
+  width: 60%;
+}
+
+.track-img-dimensions-justice {
+  width: 90%;
+  max-width: 260px;
+}
+
+.unselectable {
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -o-user-select: none;
+  user-select: none;
+}
+
+.UIUX {
+  margin-left: 35px;
+  margin-top: 25px;
+}
+
+.svg-mask {
```

Comment:
Can we place these classes inside prize tracks just so we can reduce naming collisions